Dockerfile

```diff
@@ -1,3 +1,8 @@
+FROM node:10.14.2 as builder
```

Comment:
Maybe `node:10` instead? Seem more future proof, no?

Reply:
Okay, I feel `node:10` is easy to use under development